database/crudOperations.js

```diff
@@ -0,0 +1,98 @@
+//need to import models into this file
+const {summaries, hosts} = require('./postgresSchema.js');
+
+const insertNewSummaryInfo = async (summaryInfo, cb) => {
+  //should recieve json
+  //summaryBody should include everything except hostName
+  let summaryBody = {...summaryInfo};
+  delete summaryBody.hostName;
+  //hostBody should include only hostName
+  let hostBody = {};
+  hostBody.hostName = summaryInfo.hostName;
+  // //first need to add hostName to hosts model
+  await hosts.create(hostBody)
+   .then((host) => {
```

Comment:
just for future reference, syntactically, arrow functions with single param don't need parentheses.